src/components/capture/creation/index.tsx

```diff
@@ -141,10 +141,11 @@ function NewCaptureModal() {
                 setFormSubmitting(true);
                 setFormSubmitError(null);
                 setActiveStep(Steps.WAITING_FOR_DISCOVER);
-                axiosInstance
-                    .post(links.discovery, spec.data)
+
+                discoveryEndpoint
+                    .create(links.discovery, spec.data)
                     .then((response) => {
-                        setCatalogResponse(response.data.data.attributes);
+                        setCatalogResponse(response.data.attributes);
```

Comment:
This is basically the core change of removing AXIOS. You now call a specific endpoint and the promise return no longer requires the `.data.data` approaching to getting data.